.github/workflows/ci.yaml

```diff
@@ -0,0 +1,321 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  merge_group:
+  pull_request:
+    branches:
+      - "main"
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+"
+  workflow_dispatch:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-20.04 
+    steps:
+    - name: Cancel Previous Runs
```

Comment:
this is deprecated, see https://github.com/metallb/metallb/pull/2116

Reply:
this is still using `cancel-workflow-action`, did it create problems like the other bumps?

Reply:
No, I forgot to remove it, now it's updated.